binstar_client/commands/config.py

```diff
@@ -150,13 +150,12 @@ def main(args):
 def add_parser(subparsers):
     description = 'Anaconda client configuration'
     parser = subparsers.add_parser('config',
-                                      help=description,
-                                      description=description,
-                                      epilog=__doc__,
-                                      formatter_class=RawDescriptionHelpFormatter
-                                      )
+                                   help=description,
+                                   description=description,
+                                   epilog=__doc__,
+                                   formatter_class=RawDescriptionHelpFormatter)
 
-    parser.add_argument('--type', default=yaml.safe_load,
+    parser.add_argument('--type', default=str,
```

Comment:
Any Py2/Py3 problems that may arise from that `str`?

Reply:
Good point. I'll check it out.

Reply:
Fixed.